doc/release/technotes/README.auxIO.rst

```diff
@@ -12,8 +14,13 @@ systems simultaneously.
 Dependencies
 ------------
 
-For each AIO system described here, there is a corresponding README.<system> in
-this directory detailing how to install and use them.
+Each AIO system is typically described in the support module for it.
+See:
+
+ * :mod:`HDFS`
```

Comment:
If it becomes a problem, feel free to refer to the HDFS module some other way. I do this :mod: thing in a few other READMEs too. I had to put .. default-domain:: chpl at the top of the file to get it to work.